packages/camera/camera_avfoundation/ios/Classes/CameraPlugin.m

```diff
@@ -299,6 +299,14 @@ - (void)handleCreateMethodCall:(FlutterMethodCall *)call
   });
 }
 
+// Returns integer value if provided and positive, or nil if not specified or negative.
+NSNumber *toPositiveOrNull(NSNumber *number) {
```

Comment:
Null and nil are not the same thing in Obj-C, so this name is misleading.

Reply:
Arg changed to `id` and renamed to...Nil